File: app/src/main/java/fr/scootop/data/model/services/LocationWearable.java
package fr.scootop.data.model.services;

import fr.scootop.data.model.tools.Match;
import fr.scootop.data.model.tools.Team;
import fr.scootop.data.model.user.domain.Player;

public class LocationWearable {
	
    private Long id;
    
    
    private Player player;
    
    private Team team;
    
    
    @JsonIgnoreProperties("wearables")
    private Match match;
    
    
    private Integer distanceRun;
    
    
    private Integer distancePlay;
    
    
    private Integer vMax;
    
    
    private Integer vMaxWithBall;
    
    
    private Integer ballPlay;
    
    
    private Integer looseBall;
    
    
    private Integer ballRecovered;
    
    
    private Integer tackle;
    
    
    private Integer foulSuffered;
    
    
    private Integer failPass;
    
    
    private Integer assist;
    
    
    private Integer shortPass;
    
    
    private Integer head;
    
    
    private Integer headOffensive;
    
    
    private Integer headDefensive;
    
    
    private Integer shots;
    
    
    private Integer shotOnTarget;
    
    
    private Integer shotOffTarget;
    
    
    private Integer longShot;
    
    
    private Integer successSkill;
    
    
    private Integer failSkill;
}

File: app/src/main/java/fr/scootop/data/model/services/LocationWearable.kt
package fr.scootop.data.model.services

import android.os.Parcelable
import fr.scootop.data.model.tools.Match
import fr.scootop.data.model.tools.Team
import fr.scootop.data.model.user.domain.Player
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LocationWearable (
    var id: Long? = null,
    var player: Player? = null,
    var team: Team? = null,
    var match: Match? = null,
    var distanceRun: Int? = null,
    var distancePlay: Int? = null,
    var vMax: Int? = null,
    var vMaxWithBall: Int? = null,
    var ballPlay: Int? = null,
    var looseBall: Int? = null,
    var ballRecovered: Int? = null,
    var tackle: Int? = null,
    var foulSuffered: Int? = null,
    var failPass: Int? = null,
    var assist: Int? = null,
    var shortPass: Int? = null,
    var head: Int? = null,
    var headOffensive: Int? = null,
    var headDefensive: Int? = null,
    var shots: Int? = null,
    var shotOnTarget: Int? = null,
    var shotOffTarget: Int? = null,
    var longShot: Int? = null,
    var successSkill: Int? = null,
    var failSkill: Int? = null
) : Parcelable